association_backend/tests/prepare_db.py

```diff
@@ -0,0 +1,30 @@
+# isort: off
+import asyncio
+import os
+import sys  # noqa
+
+from association.db import get_engine
+from association.domain.entities.subscription_entities import mapper_registry
+from association.settings import DATABASE_URL
+from db_utils import create_database, database_exists
+
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))  # noqa
```

Comment:
```suggestion
# isort: off
import asyncio
import os
import sys  # noqa

sys.path.append(os.path.join(os.path.dirname(__file__), ".."))  # noqa

from association.db import get_engine
from association.domain.entities.subscription_entities import mapper_registry
from association.settings import DATABASE_URL
from db_utils import create_database, database_exists

```

this should be more correct, the `sys.path.append` should happen before 

```
from db_utils import create_database, database_exists
```

otherwise it can't find `db_utils`?